specificatie/genereervariant/openapi.yaml

```diff
@@ -23,15 +23,15 @@ paths:
       tags:
       - ingeschrevenpersonen
       description: |
-        Het ophalen van een collectie ingeschreven personen inclusief het verblijfsadres, ouders, partners, kinderen en reisdocumenten. Het betreft alleen actuele gegevens van de betreffende ingeschreven personen. Ten minste één van de volgende combinaties van parameters moet zijn opgenomen:
+        Het ophalen van een collectie ingeschreven personen inclusief het verblijfsadres, ouders, partners, kinderen en reisdocumenten. Het betreft alleen actuele gegevens van de betreffende ingeschreven personen. Ten minste een van de volgende combinaties van parameters moet zijn opgenomen:
 
 
         1.  Persoon
             -  geboorte__datum
-            -  naam__geslachtsnaam (minimaal 2 karakters, [wildcard toegestaan](https://github.com/VNG-Realisatie/Bevragingen-ingeschreven-personen/blob/master/features/wildcard.feature) )
+            -  naam__geslachtsnaam (minimaal 2 karakters, [wildcard toegestaan](https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.1.0/features/wildcard.feature) )
```

Comment:
waarom wordt hier naar de raw content verwezen? Doen we ook niet voor andere features. Wanneer je naar de blob variant verwijst (zoals hier voorheen ook werd gedaan) krijg je de feature mooi met kleurtjes te zien.

Reply:
Aangepast